pkg/metrics/metrics.go

```diff
@@ -50,6 +50,29 @@ var (
 		Help:      "Count of all endpoint regenerations that have completed, tagged by outcome",
 	},
 		[]string{"outcome"})
+
+	// Policies
+
+	// NumPolicies is the number of policies loaded into the agent
+	NumPolicies = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "policies",
```

Comment:
I suggest to just name this `policy` to be consistent, we never call it `policies`.

Reply:
done. I also renamed endpoint and event to be consistent all around :)